apps/codeforafrica/env.template

```diff
@@ -1,4 +1,7 @@
 NEXT_PUBLIC_IMAGE_DOMAINS=
+NEXT_PUBLIC_APP_LOGO_URL=https://res.cloudinary.com/code-for-africa/image/upload/v1656078512/codeforafrica/logos/CfA_logo_BW_twlmo6.png
```

Comment:
This shouldn't be an env var... Or why is it one?

Reply:
there's a possibility of the URL changing e.g if we decide to use a different logo. No?

Reply:
@kilemensi  does ☝️ qualify for an env variable?